### ISSUE TITLE
Surround sound now working, only outputting through front left and front right

### ISSUE THREAD

**Issue by silentsudin** [ublue-os/bazzite#1877]
### Describe the bug

On Bazzite 41, I am not getting surround sound audio. I have a 5700x3d and 7800 XT, and am using Displayport to hdmi converter (I have also tried a normal hdmi cable).

When I set the output to 5.1 or 7.1, I am only getting sound from the front 2 speakers, I do not get sound from any other speakers. 

### What did you expect to happen?

I expected 2 + 2 to equal 4, but instead 2 + 2 equaled 6!

### Output of `rpm-ostree status`

```shell

```

**Comment by silentsudin** [comment]
I rebased back to bazzite stable 40 and I still am having this issue

**Comment by karimElmougi** [comment]
I am experiencing this issue as well, on the latest stable build.

**Comment by felixhirschfeld** [comment]
I just re-installed Bazzite completely and now I have the same problem. When I first installed Bazzite in February I had no issues with this.

**Comment by apalazzi** [comment]
I have a similar issue: pc connected to TV which is connected to 5.1 home theatre. Output is correctly detected as 5.1 but I only get sound from front speakers, even with the testing applet.
Has anyone figured out the issue and have a solution?

**Comment by okeribok** [comment]
I have this problem too. Both 5.1 and 7.1 output only front left and right, as tested in desktop mode settings> audio and experienced in games in steam mode; the sureound is recognized, but only left and right get output; sounds really weird without center and the surrounds. Continuing issue, has been like this for months. Ryzen ai max 395+

**Comment by apalazzi** [comment]
Hi, see this post on reddit, it worked for me: https://www.reddit.com/r/Bazzite/comments/1rp45p9/guide_true_51_surround_sound_ac3_on_bazzite_via/

**Comment by addmix** [comment]
> Hi, see this post on reddit, it worked for me: https://www.reddit.com/r/Bazzite/comments/1rp45p9/guide_true_51_surround_sound_ac3_on_bazzite_via/

This workaround worked for me. I think this issue needs to be reopened.

**Comment by apalazzi** [comment]
Absolutely, the issue is still present and there's also a fix available: no reason to not fix it.
